SoomlaiOSProfile/SoomlaSerializableObject.h

```diff
@@ -0,0 +1,31 @@
+//
+// Created by Eugene Butusov on 21/11/15.
+// Copyright (c) 2015 Soomla. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "UserProfileUtils.h"
+
+
+@interface SoomlaSerializableObject : NSObject {
```

Comment:
@borisspektor, I did it to avoid code duplication, similar how it did in LevelUp and Store modules (but there is SoomlaEntity class, I can move this implementation here.